velox/exec/GroupingSet.cpp

```diff
@@ -656,9 +657,10 @@ bool GroupingSet::getDefaultGlobalGroupingSetOutput(
   // aggregates.
   for (auto i = firstAggregateCol; i < outputType->size(); i++) {
     auto resultAggregateColumn = result->childAt(i);
+    resultAggregateColumn->resize(numGroupingSets);
```

Comment:
This is the change, right? 

I believe these will no longer be needed after a change to RowVector::resize to resize children that @kgpai is working on.

Reply:
@mbasmanova : Yes, that is the change.

The change @kgpai is doing will be great. Its very error prone to add such code for resizing all children each time.